django/cantusdb_project/main_app/tests/make_fakes.py

```diff
@@ -263,28 +265,56 @@ def make_fake_sequence(source=None, title=None, incipit=None, cantus_id=None) ->
     return sequence
 
 
-def make_fake_source(published=None, title=None, segment=None, segment_name=None, siglum=None, description=None, summary=None, provenance=None, full_source=None, rism_siglum=None, indexing_notes=None) -> Source:
+def make_fake_source(
+        published:Optional[bool]=None,
+        title:Optional[str]=None,
+        segment_name:Optional[str]=None,
+        segment:Optional[Segment]=None,
+        siglum:Optional[str]=None,
+        rism_siglum:Optional[RismSiglum]=None,
+        description:Optional[str]=None,
+        summary:Optional[str]=None,
+        provenance:Optional[Provenance]=None,
+        century:Optional[Century]=None,
+        full_source:Optional[bool]=None,
+        indexing_notes:Optional[str]=None,
+    ) -> Source:
     """Generates a fake Source object."""
     # The cursus_choices and source_status_choices lists in Source are lists of
     # tuples and we only need the first element of each tuple
 
+    if published is None:
```

Comment:
I get what you're saying. I like the way things are currently because all the parameters are present, in the same order - the parameters declared at the top of the function, the series of `if` statements here, and the arguments to `Chant.objects.create()` - and they're all handled in the same way. As well, I figure tests are a place where I'd tend to prioritize clarity/consistency over efficiency.

With that said, I could make this nearly as clear by replacing this `if` block with a comment (a change that will, at the very least, prevent future developers from thinking "why was this not done with a default argument in the first place?"). (I should also add placeholder comments for `century`, `provenance`, etc. among the `.create()` arguments below.) So sure, I can make this change.